src/encoded/schemas/file_format.json

```diff
@@ -127,22 +127,106 @@
                 "description": "Item class name (e.g. FileFastq)",
                 "type": "string"
             }
+        },
+        "assay_types": {
+            "title": "Assay types",
+            "description": "Assays that make use of this file format",
+            "type": "array",
+            "items": {
+                "title": "Assay or Category",
+                "type": "string",
+                "suggested_enum": [
+                    "Sequencing",
+                    "Microscopy",
+                    "Hi-C",
+                    "SPRITE",
+                    "RNA-seq",
+                    "Generic"
+                ]
+            }
+        },
+        "file_content": {
+            "title": "File content",
+            "description": "Type of information this file is used for",
+            "type": "string",
+            "suggested_enum": [
+                "chromosome sizes",
+                "continuous genomic data",
+                "genomic features",
+                "alignments",
+                "contact list",
+                "contact matrix",
+                "image",
+                "intermediate file",
+                "multi-contact read clusters",
+                "other",
+                "reads",
+                "sequences",
+                "SPT results",
```

Comment:
Might this be better as something more general like 'particle position or trajectory results'?

Reply:
Since this is meant to appear in a table column 'particle position or trajectory results' would be too long to be useful I think but I can think about whether I can come up with an alternative.